### ISSUE TITLE
appending to sys.path

### ISSUE THREAD

**Issue by rsgalloway** [rsgalloway/envstack#8]
currently the default `stack.env` file sets `PYTHONPATH` but `envstack.init()` does not append to or modify `sys.path`.

works:
```bash
$ envstack -- python
>>> import foobar
>>> 
```

does not work:
```python
>>> envstack.init()
>>> import foobar
ModuleNotFoundError: No module named 'foobar'
```

should envstack.init() parse $PYTHONPATH and modify sys.path in the same way that sitecustomize.py does in [siteconf](https://github.com/rsgalloway/siteconf)? or just couple envstack with siteconf in some way?

**Comment by rsgalloway** [comment]
sitecutomize only loads when Python starts, it won't modify sys.path later, nor does Python include a built-in module that specifically parses and modifies sys.path directly based on the PYTHONPATH environment variable.

```python
# parse PYTHONPATH and add directories to sys.path
pythonpath = os.environ.get("PYTHONPATH")
if pythonpath:
    for path in pythonpath.split(os.pathsep):
        if path not in sys.path:
            sys.path.insert(0, path)
```

**Comment by rsgalloway** [comment]
working on this in branch issue8/init_sys_path

**Comment by rsgalloway** [comment]
resolved in version 0.6.3
